nil pointer dereference using image with index colorspace

Error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x765be6]

goroutine 1 [running]:
github.com/signintech/gopdf.(*DeviceRGBObj).protection(0xc42017e000, 0xc42017e020)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/device_rgb_obj.go:19 +0x26
github.com/signintech/gopdf.(*DeviceRGBObj).write(0xc42017e000, 0x8cc080, 0xc42017e020, 0x12, 0xc420069ab8, 0x1)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/device_rgb_obj.go:33 +0x161
github.com/signintech/gopdf.(*GoPdf).compilePdf(0xc4200dc480, 0x8cbbc0, 0xc4200dc600, 0xc420069b50, 0x492684)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/gopdf.go:422 +0x270
github.com/signintech/gopdf.(*GoPdf).GetBytesPdfReturnErr(0xc4200dc480, 0x0, 0xc42044c050, 0x0, 0xc420476000, 0xc420069b90)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/gopdf.go:453 +0x88
github.com/signintech/gopdf.(*GoPdf).GetBytesPdf(0xc4200dc480, 0x1, 0x8d1d80, 0xc420476000)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/gopdf.go:459 +0x2f
github.com/signintech/gopdf.(*GoPdf).WritePdf(0xc4200dc480, 0xc42044c050, 0x4d)
	/home/mspindelhirn/go/src/github.com/signintech/gopdf/gopdf.go:383 +0x2b
```

Error occurs when adding an image with indexed colorspace.

Cause:
The `DeviceRGBObj` returned doesnt get a `getRoot` function assigned to return the `*GoPdf` root instance. Later that root instance is dereferenced throwing the error above

--- a/gopdf.go
+++ b/gopdf.go
@@ -254,6 +254,9 @@ func (gp *GoPdf) ImageByHolder(img ImageHolder, x float64, y float64, rect *Rect
 			if err != nil {
 				return err
 			}
+			dRGB.getRoot = func() *GoPdf {
+				return gp
+			}
 			imgobj.imginfo.deviceRGBObjID = gp.addObj(dRGB)
 		}
 

